apps/judicial-system/web/src/utils/titleForCase/titleForCase.spec.ts

```diff
@@ -0,0 +1,179 @@
+import { createIntl } from 'react-intl'
+
+import { CaseState } from '@island.is/judicial-system/types'
+import { TempCase as Case } from '@island.is/judicial-system-web/src/types'
+import { CaseType } from '@island.is/judicial-system-web/src/graphql/schema'
+
+import { titleForCase } from './titleForCase'
+
+describe('titleForCase', () => {
+  const formatMessage = createIntl({ locale: 'is', onError: jest.fn })
+    .formatMessage
+  const fn = (theCase: Case) => titleForCase(formatMessage, theCase)
+
+  test('should handle rejected investigation case', () => {
+    const theCase = {
+      state: CaseState.REJECTED,
+      type: CaseType.BODY_SEARCH,
+    } as Case
+    const res = fn(theCase)
+    expect(res).toEqual('Kröfu um rannsóknarheimild hafnað')
```

Comment:
Will these tests fail if we change the strings in Contentful? If so I wonder if there might be a better way to conduct these kinds of tests that doesn't rely directly on the strings themselves. Just food for thought 🤗